.gitignore

```diff
@@ -3,3 +3,4 @@ secretKey.json
 **/pgdata
 output.txt
 **/.DS_Store
+.env
```

Comment:
https://github.com/Quran-Journey/backend/tree/64-firebase/target
What's the use of this folder? is it generated by firebase?
If it is, let's make sure it's not merged into master by adding it to the gitignore (you'll have to delete it and push to the branch before adding it to gitignore)